monitoring/check_pis.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+pswd="piman"
```

Comment:
it's a bad idea to put passwords in source. can you get it from the command line? perhaps $1